### PR TITLE
More checkout polishing

### PR DESCRIPTION
**Pull request by nlindblad**

### CODE REVIEW

File: app/views/fragments/checkout/reviewDetails.scala.html [hunk 1]
```diff
@@ -34,7 +34,7 @@
             <div class="review-details__list">
                 <ul class="o-unstyled-list">
                     <li class="js-checkout-review-delivery-address sessioncamhidetext"></li>
-                    <li><span class="review-details__detail">Week of first paper:</span> <span class="js-checkout-review-delivery-start-date"></span></li>
+                    <li><span class="review-details__detail">Voucher book start date:</span> <span class="js-checkout-review-delivery-start-date"></span></li>
```
Comment (tomverran) [inline app/views/fragments/checkout/reviewDetails.scala.html]: Hmm does this get used for delivery too?

Comment (tomverran) [inline app/views/fragments/checkout/reviewDetails.scala.html]: it does not!

**Comment by tomverran** [comment]
👍

File: app/views/support/PlanOps.scala [hunk 2]
```diff
@@ -50,7 +50,7 @@ object PlanOps {
 
     def hasDigitalPack: Boolean = in.products.seq.contains(Digipack)
 
-    def phone: String = "+44 (0) 330 333 6767"
+    def phone: String = "0330 333 6767"
```
Comment (paulbrown1982) [inline app/views/support/PlanOps.scala]: There are multiple instances of the phone number, throughout a multitude of templates and view classes. We should probably write a story to tidy that up into one helper class with the methods: phoneInternational and phoneUk

Comment (nlindblad) [inline app/views/support/PlanOps.scala]: I created a card for it.

**Comment by paulbrown1982** [comment]
👍

**Pull request merged**